app/crawler/crawler.py

```diff
@@ -2,33 +2,12 @@
 Crawls an XML/HTML document. It can extract urls for further crawling and
 optionally store the documents it has visited.
 '''
```

Comment:
It's a module description. Flake8 complains about it in every file I neglect to do it